src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -307,6 +388,7 @@ private String buildOAuthRedirectUrl() {
      * This is where the user comes back to at the end of the OpenID redirect ping-pong.
      */
     public HttpResponse doFinishLogin(StaplerRequest request) throws IOException {
+    	LOGGER.fine("Realm do finish " + request);
```

Comment:
I'll add the if checks, but some background (I had specifically omitted them):
- there was actually a fair amount of debate back with our former employer's app server team and its performance group (@bparees sanity checked me on this, recalls this as well) wrt JIT optimizations, 2 function calls vs. one when trace was enabled, cluttering the code for non-critical performance paths, along with the points you rose)
- There general opinion leaned toward omitting the if checks, but I think it is fair to say that was not conclusive

Aside from adding the if check, I'm also reducing the amount of trace points (in hindsight a good portion of it stemmed from being new to the code and oauth flow and trying to get things working for the first time) and will clean up the prefixing.